### ISSUE TITLE
Custom datetime inputFormat and separator symbol input

### ISSUE THREAD

**Issue by jaroslavlibal** [RobinHerbots/Inputmask#2023]
I would like to have the same behavior in the 4.x branch which I used to have in the 3.x branch version. That means that when I use the custom datetime inputFormat like:

```
$('.inputmask').inputmask("datetime",
    {
        inputFormat: "dd.mm.yyyy",
        min: "01.01.1900",
        max: "31.12.2099",
    }
);
```

then the "." separator symbol input should result in "skipping" to the next input group.

Eg. "1.1.1950" input should result in "01.01.1950" value, but the mask is waiting for the second numeric input instead. It works when the input is "4.2.1950" because there are no other valid posibilities then "04.02.1950".

Is there any chance to react to the spearator value input, please? Thanks!

**Comment by jaroslavlibal** [comment]
Same issue still persists in the 5.0.0-beta.163

When the user's input is `1.1.` then the value is `01.1` and the inputmask is waiting for the second number from the month.

Just realized that it is "caused by" the `min` option. It works correctly with this init:

    $('.inputmask').inputmask("datetime",
        {
            inputFormat: "dd.mm.yyyy",
            max: "31.12.2099",
        }
    );

Any idea, please?

**Comment by RobinHerbots** [comment]
@jaroslavlibal ,

This is fixed in versdion 5.0.7-beta.12
